code/modules/mob/living/simple_animal/friendly/corgi.dm

```diff
@@ -19,6 +19,7 @@
 	response_help  = "pets"
 	response_disarm = "bops"
 	response_harm   = "kicks"
+	mob_size = MOB_SIZE_HUMAN

```

Comment:
So I either needed to add a new size type and shift everything, or say that dogs are bigger than small and smaller than large